src/views/InvolvementProfile/components/Membership/components/AdminCard/index.js

```diff
@@ -27,6 +27,16 @@ const headerStyle = {
   padding: '10px',
 };
 
+const stripDomain = (input, domain = '@gordon.edu') => {
```

Comment:
Do we want to move this to `utils.ts` so that it can be re-used in other places more easily?

Reply:
Perhaps that makes sense. I often forget that we have a utils service.